### PR TITLE
🇨🇳 Add Simplified Chinese support

### PR DESCRIPTION
**Pull request by yimingtang**
Hi, Sam. Here's Simplified Chinese translation for Redacted.

Note:

Not sure if there are suitable Chinese words for "redacted" and "redaction". 

* The common translation "纂" I found is rarely used in daily life. It only exits in literary writings. 
* The other translation "编辑" is the same as "edit".

I think "redacted" and "redaction" are highly related to your app name, so I leave them untranslated. It seems okay to keep the original words.

### CODE REVIEW

File: zh-Hans.lproj/Shared.strings [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+"SHARE" = "分享";
+"CLEAR" = "清除";
+"CLEAR_IMAGE" = "清除图片";
+"SAVE" = "保存";
+"EXPORT" = "导出…";
+
+/* Edit menu */
+"COPY_IMAGE" = "复制图片";
+"PASTE_IMAGE" = "粘贴图片";
+"SELECT_ALL_REDACTIONS" = "选择所有 Redactions";
```
Comment (yimingtang) [inline zh-Hans.lproj/Shared.strings]: Keep "Redactions" untranslated

File: zh-Hans.lproj/Shared.strings [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+/* Undo actions */
+"INSERT_REDACTION" = "插入 Redaction";
+"INSERT_REDACTIONS" = "插入 Redactions";
+"DELETE_REDACTION" = "删除 Redaction";
+"DELETE_REDACTIONS" = "删除 Redactions";
+
+/* About */
+"VERSION_FORMAT" = "版本 %@";
+"CREDITS" = "由 Nothing Magical 在加利福尼亚设计和开发。";
```
Comment (yimingtang) [inline zh-Hans.lproj/Shared.strings]: Also keep the original name

**Comment by soffes** [comment]
Thanks so much! This looks great so far!

I just pushed some more strings a few minutes ago. I think this is all of the final stuff for iOS 1.0. Mind adding the additional strings?

File: zh-Hans.lproj/Shared.strings (hunk 1 above)
Comment (soffes) [reply zh-Hans.lproj/Shared.strings]: Is there not a good Chinese word for this? I'm fine to leave it in English if you think that's best. I'm not treating it like the app name but just the generic word so feel free to translate it if you want to!

**Comment by yimingtang** [comment]
😎 Done.

**Comment by soffes** [comment]
Thank you!!

**Pull request merged**